frontend/api/configuration.ts

```diff
@@ -90,11 +90,10 @@ export class Configuration {
         this.basePath = param.basePath;
         this.serverIndex = param.serverIndex;
         this.baseOptions = {
```

Comment:
Si je comprendds bien, vue que headers est juste un spread de baseOptions.headers, je pense que ça ne veut plus la peine de spread les deux

```suggestion
        this.baseOptions = param.baseOptions
```